easybuild/easyconfigs/x/X11/X11-20170314-foss-2017b.eb

```diff
@@ -0,0 +1,137 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20170314'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [
+    XORG_LIB_SOURCE,
+    XORG_PROTO_SOURCE,
+    'http://xcb.freedesktop.org/dist/',
+    'http://xkbcommon.org/download/',
+    'http://cgit.freedesktop.org/xorg/util/macros/snapshot',
+    XORG_DATA_SOURCE + '/xkeyboard-config',
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('fontconfig', '2.12.6', '-libpng-1.6.34'),
+    ('freetype', '2.8.1', '-libpng-1.6.34'),
```

Comment:
It's probably easier to stick to `fontconfig` `2.12.4` and `freetype` `2.8` to avoid that these minor (and potentially irrelevant bumps) spread like a virus?